R/lazyframe__lazy.R

```diff
@@ -338,6 +338,7 @@ LazyFrame_set_optimization_toggle = function(
 #' @description `$collect()` performs the query on the LazyFrame. It returns a
 #' DataFrame
 #' @inheritParams LazyFrame_set_optimization_toggle
+#' @param ... Ignored.
```

Comment:
I don't understand why you add this here. I get that the `as_polars()` methods must have these dots but nothing forces us to pass them to `$collect()` or `fetch()` in the method for `LazyFrame`

Reply:
The intent is to force these arguments to be specified by name.